src/plugins/timeConductor/independent/Mode.vue

```diff
@@ -0,0 +1,216 @@
+            activeClock = this.openmct.time.clock();
+        }
+
+        if (activeClock !== undefined) {
+            //Create copy of active clock so the time API does not get reactified.
+            activeClock = Object.create(activeClock);
+        }
+
+        return {
+            selectedMode: this.getModeOptionForClock(activeClock),
+            modes: []
+        };
+    },
+    mounted: function () {
+        if (this.mode) {
+            this.setViewFromClock(this.mode === 'fixed' ? undefined : this.mode);
+        }
+
+        this.openmct.time.on('clock', this.setViewFromClock);
+    },
+    destroyed: function () {
+        this.openmct.time.off('clock', this.setViewFromClock);
+    },
+    methods: {
+        showModesMenu() {
+            const elementBoundingClientRect = this.$refs.modeButton.getBoundingClientRect();
+            const x = elementBoundingClientRect.x;
+            const y = elementBoundingClientRect.y;
+
+            const menuOptions = {
+                menuClass: 'c-conductor__mode-menu',
+                placement: this.openmct.menus.menuPlacement.TOP_RIGHT
+            };
+            this.openmct.menus.showSuperMenu(x, y, this.modes, menuOptions);
+        },
+
+        getMenuOptions() {
+            let clocks = [{
+                name: 'Fixed Timespan',
+                timeSystem: 'utc'
+            }];
+            let activeClock = this.openmct.time.clock();
+            if (activeClock !== undefined) {
+            //Create copy of active clock so the time API does not get reactified.
+                activeClock = Object.assign({}, {
+                    name: activeClock.name,
+                    clock: activeClock.key,
+                    timeSystem: this.openmct.time.timeSystem().key
+                });
+
+                clocks.push(activeClock);
+            }
+
+            return clocks;
+        },
+        loadClocks() {
+            let clocks = this.getMenuOptions()
+                .map(menuOption => menuOption.clock)
+                .filter(isDefinedAndUnique)
+                .map(this.getClock);
+
+            /*
+         * Populate the modes menu with metadata from the available clocks
+         * "Fixed Mode" is always first, and has no defined clock
+         */
+            this.modes = [undefined]
+                .concat(clocks)
+                .map(this.getModeOptionForClock);
+
+            function isDefinedAndUnique(key, index, array) {
+                return key !== undefined && array.indexOf(key) === index;
+            }
+        },
+
+        getModeOptionForClock(clock) {
+            if (clock === undefined) {
+                const key = 'fixed';
+
+                return {
+                    key,
+                    name: 'Fixed Timespan',
+                    description: 'Query and explore data that falls between two fixed datetimes.',
+                    cssClass: 'icon-tabular',
+                    callBack: () => this.setOption(key)
```

Comment:
if you are using this in `this.openmct.menus.showMenu(`, we changed `callback` to `onItemClicked`

Reply:
Done